src/sentry/integrations/discord/client.py

```diff
@@ -115,17 +116,32 @@ def track_response_data(
         resp: Response | None = None,
         extra: Mapping[str, str] | None = None,
     ) -> None:
+        discord_code = ""
+        code_message = ""
+
+        try:
+            json_data = resp.json()
+            if isinstance(json_data, dict):
+                discord_code = json_data.get("code")
+                if discord_code in DISCORD_ERROR_CODES:
+                    code_message = DISCORD_ERROR_CODES[discord_code]
+
+        except Exception:
+            pass
+
         is_ok = code in {
             status.HTTP_200_OK,
             status.HTTP_201_CREATED,
             status.HTTP_202_ACCEPTED,
             status.HTTP_204_NO_CONTENT,
         }
-        include_in_slo = code not in {
-            status.HTTP_429_TOO_MANY_REQUESTS,
-            status.HTTP_403_FORBIDDEN,  # Is user error
-        }
 
+        # These are excluded since they are not actionable from our side
+        discord_server_issues = {
+            status.HTTP_500_INTERNAL_SERVER_ERROR,
+            status.HTTP_503_SERVICE_UNAVAILABLE,
```

Comment:
I don't quite agree with this logic. If Discord goes down, we need to be aware of it because it means our integration isn't working for all our customers. 

The action item would be to report it on status.sentry.io.

Reply:
+1

I don't think we should be trying to ignore anything at the root status code level. We need to be alerted that the integration is down, and our SLOs help us with that, which is powered by the reporting